codec-redis/src/main/java/io/netty/handler/codec/redis/RefCountedArrayRedisMessage.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.util.AbstractReferenceCounted;
+import io.netty.util.ReferenceCountUtil;
+import io.netty.util.internal.ObjectUtil;
+import io.netty.util.internal.StringUtil;
+
+import java.util.List;
+
+public class RefCountedArrayRedisMessage extends AbstractReferenceCounted implements ArrayRedisMessage {
+
+    private final List<RedisMessage> children;
+
+    public RefCountedArrayRedisMessage(List<RedisMessage> children) {
+        ObjectUtil.checkNotNull(children, "children cannot be null");
+        this.children = children; // do not retain here. children are already retained when created.
```

Comment:
merge these two lines. And also replace "children cannot be null" with "children" as this is the style we usually use in netty.